Installing-Oppia-using-Docker.md

```diff
@@ -105,16 +106,22 @@ Download and install the latest version of Docker Desktop from the [official Doc
    This command launches the Oppia development server in offline mode, and you can continue to perform your tasks as usual.
    > NOTE: Ensure that you have already built Oppia for the first time before running this command. If not, run `make build` first (with internet connection), which downloads all the necessary third-party libraries, python dependencies, and the other services required for the Oppia development server.
 
-5. **Stop the Local Development Server**: To stop the local development server, execute the following command:
+6. **Stop the Local Development Server**: To stop the local development server, execute the following command:
 
    ```
    make stop
    ```
 
    This command stops the Oppia development server and shuts down the Docker containers running the Oppia development server.
 
+
+**Note for Developers: Allocating Resources to Docker Desktop**
+For systems with 8GB RAM:
```

Comment:
Please explain how a developer can find out how much RAM they have on their system. 

Instead of putting this at the end, can you make this more of a pre-check in the instructions?